From de9cdf80b5d61bf4182bad76b7f318b57321999e Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Fri, 6 Mar 2009 18:00:51 +0800
Subject: [PATCH] ENGR00108994 MX35: I2C bus can't work when rmmod and re-insmod camera module

"VIOHI" is also used as I2C power source, it can't be disabled
by other modules

Signed-off-by: Wallace Wang <r59996@freescale.com>
---
 arch/arm/mach-mx35/mx35_3stack.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx35/mx35_3stack.c b/arch/arm/mach-mx35/mx35_3stack.c
index 7f95edf..5547248 100644
--- a/arch/arm/mach-mx35/mx35_3stack.c
+++ b/arch/arm/mach-mx35/mx35_3stack.c
@@ -354,7 +354,7 @@ static struct mxc_tsc_platform_data tsc2007_data = {
 static struct mxc_camera_platform_data camera_data = {
 	.core_regulator = "SW1",
 	.io_regulator = "VAUDIO",
-	.analog_regulator = "VIOHI",
+	.analog_regulator = NULL,
 	.gpo_regulator = "PWGT1",
 	.mclk = 27000000,
 };
-- 
1.5.4.4

